.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### The Problem/Feature
```

Comment:
Should the user replace this with the problem or feature or write this below? add a comment(s) to make this clear to users.